index.html

```diff
@@ -0,0 +1,23 @@
+
+
```

Comment:
I kindly suggest the following changes:

Please, do not leave more than one blank line space. This is coding good practice that allows your code to look clean and clearer to the person reading it.
I kindly recommend writing your doctype declaration, ''. A doctype declaration tells the browser that the page to be rendered is written in HTML.
Please, add an HTML tag to your file and the language the page is written in. The HTML tag communicates to the web browser that everything within the opening html tag, <HTML lang="en"> and the closing html tag, </html> is HTML text.
Please add a head element. The head element stores metadata for the browser. This metadata could include information like what language the page is written in, a title for the web page, or a link to a style sheet.
Please add a meta charset tag inside your head element. HTML5 specification encourages web developers to use the UTF-8 character set, which covers almost all of the characters and symbols in the world! Please, add this line code
As best practice, please add an X-UA-Compatible meta tag. This tag allows web authors to choose what version of Internet Explorer the page should be rendered as.
Please add a viewport meta tag. The viewport is the user's visible area of a web page. HTML5 let web designers take control over the viewport, through the viewport meta tag.